src/index.html

```diff
@@ -60,7 +60,9 @@ <h1 class="profile__name">Bessie Coleman</h1>
                 <div class="card__content">
                   <h2 class="card__title"></h2>
                   <button class="card__delete-btn"></button>
+                  <div class="card__like-container">
                   <button type="button" class="card__like-btn"></button>
+                  <span class="card__like-count"></span>
                 </div>
```

Comment:
Please, add another ` </div>` here to close the `  <div class="card__like-container">`

```html

            <template id="card-template">
              <li class="card">
                <img src="" alt="" class="card__image" />
                <div class="card__content">
                  <h2 class="card__title"></h2>
                  <button class="card__delete-btn"></button>
                  <div class="card__like-container">
                    <button type="button" class="card__like-btn"></button>
                    <span class="card__like-count"></span>
                  </div>
                </div>
              </li>
            </template>
```